examples/scale/README.md

```diff
@@ -0,0 +1,5 @@
+# Basics
```

Comment:
```suggestion
# Scale
```
maybe?
This also needs to be added to the main readme inside the examples directory